docs/specification.md

```diff
@@ -101,8 +101,8 @@ type config =
 
   ; max_proposals : nat
   // ^ Determine the maximum number of ongoing proposals that are allowed in the contract.
-  ; max_votes : nat
-  // ^ Determine the maximum number of votes associated with a proposal.
+  ; max_voters : nat
+  // ^ Determine the maximum number of voters that are allowed to vote in the context of a proposal.
```

Comment:
```suggestion
  // ^ Determine the maximum number of voters that are allowed to vote on a proposal.
```
Nitpick, but this seems simpler.

Reply:
Also I think that here it's worth adding a note that specifies that voters that votes on the same proposal in both ways are counted twice, what do you think?

Reply:
Oh, I think that is restriction will appear to be a bit arbitrary. Can we count them only once? It would make `vote` operation more costly. May be we can limit the voter count to `2 * max_voters`, allowing every voter to vote both ways in they choose.

What do you think?

Reply:
Right now I have modifed the spec to say the voters are counted twice. Will change it if you think otherwise.